packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -110,12 +110,11 @@ describe(`background sync queue`, function() {
     await backgroundSyncQueue.pushIntoQueue({
       request: new Request('/__test/404'),
     });
-    try {
-      await backgroundSyncQueue.replayRequests();
+    return backgroundSyncQueue.replayRequests().then(()=>{
       throw new Error('Replay should have failed because of invalid URL');
-    } catch (err) {
-      expect(err[0].status).to.equal(404);
-    }
+    }).catch(()=>{
```

Comment:
This still will always pass. You need to have `.then(() => {}, () => {})` and throw in the first callback.